### ISSUE TITLE
No space between author first and last name because last char of first name is translated to a latex command

### ISSUE THREAD

**Issue by samuelgruetter** [retorquere/zotero-better-bibtex#1091]
Report ID: 8LNKRTAU-euc

Version: 5.1.12

`bibtex --version`: `MiKTeX-BibTeX  (MiKTeX 2.9.6840)`
`pdflatex --version` `MiKTeX-pdfTeX 2.9.6839 (1.40.19) (MiKTeX 2.9.6840)`

If I have a record whose author is "Michał Lastname" (note the special l), I get a `.bib` file with

```
  author = {Lastname, Micha\l},
```

and if I compile this with `pdflatex`, there's no space between the first and last name of the author.
I've had this problem with latex before in cases where no bibliographies were involved, and I solved it by writing `Micha{\l} Lastname` instead of just `Micha\l Lastname` in my `.tex` source, and I think zotero-better-bibtex should do that too.

**Comment by retorquere** [comment]
I'll try to put together a fix but I'll also try to find out what the specced/recommended behavior is for this kind of problem

For my own reference, a MWE that exhibits the problem:

```
\documentclass{article}

\usepackage{filecontents}

\begin{filecontents}{\jobname.bib}
  @book{Knu86,
    author = {Lastname, Micha\l},
    year = {1986},
    title = {The \TeX book},
  }
\end{filecontents}

\begin{document}

From \cite{Knu86} we see \ldots

\bibliographystyle{alpha}
\bibliography{\jobname}
\end{document}
```

which renders to

```
From [Las86] we see . . .
References
[Las86]  MichalLastname.
The TEXbook.  1986
```

**Comment by blip-bloop** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 5.1.12.3054](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.1.12.3054.xpi) ("adjust test cases for #1091")

Install in Zotero by downloading [test build 5.1.12.3054](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.1.12.3054.xpi), opening the Zotero "Tools" menu, selecting "Add-ons", open the gear menu in the top right, and select "Install Add-on From File...".

**Comment by retorquere** [comment]
Can you try 3054?

**Comment by samuelgruetter** [comment]
Wow, that's cool build and test automation :+1: 
So I installed 3054 and it works as it should on this example, thanks for fixing this!

**Comment by retorquere** [comment]
Thanks for the confirmation, I did add a test to cover this issue to my test suite, but it's always nice to be sure indeed your problem is solved. I've merged it into the mainline and will release a new version somewhere today after I've merged #1092.
